tests/features/licenseexport_cdx.feature

```diff
@@ -36,38 +36,158 @@ Scenario: User Downloads license information for CycloneDX SBOM from SBOM Explor
+	Then name column should contain the value of SBOM name from metadata.component.name field from SBOM json
 	And namespace column should be empty
-	And group column should contain the value of component.group field from SBOM json
-	And version column should contain the value of component.version field from SBOM json
+	And group column should contain the value of metadata.component.group field from SBOM json
+	And version column should contain the value of metadata.component.version field from SBOM json
 	And package reference column should contain the value of components.purl from SBOM json
 	And license column should contain the value of components.license.id field from SBOM json
-	And license name column should contain the value of components.license.name field from SBOM json
+	And license name column should be empty
+	And license expression column should be empty
+	And alternate package reference column should be empty
+
+Scenario: Verify the license information for a package on the CycloneDX SBOM with single license id with alternate package reference
+	# Test data https://drive.google.com/drive/folders/1Z6y6gMegutBeUuc_8LkpYxKeGz_KYG9H?usp=drive_link
+	# sbom - tc_1730_license_escape.json package - pkg:pkg:npm/%40gradio/accordion@0.3.4
+	Given User is on SBOM license information file
+	When User selects a package with Single license id with cpe information
+	Then name column should contain the value of SBOM name from metadata.component.name field from SBOM json
+	And namespace column should be empty
+	And group column should contain the value of metadata.component.group field from SBOM json
+	And version column should contain the value of metadata.component.version field from SBOM json
+	And package reference column should contain the value of components.purl from SBOM json
+	And license column should contain the value of components.license.id field from SBOM json
+	And license name column should be empty
+	And license expression column should be empty
+	And alternate package reference column should contain the value of CPE from components.cpe field from SBOM json
+
+Scenario: Verify the license information for a package on the CycloneDX SBOM with single license name
+	# Test data https://drive.google.com/drive/folders/1Z6y6gMegutBeUuc_8LkpYxKeGz_KYG9H?usp=drive_link
+	# sbom - cdx_sbom.json package - pkg:maven/org.bouncycastle/bcpkix-jdk15on@1.68?type=jar
+	Given User is on SBOM license information file
+	When User selects a package with Single license name
+	Then name column should contain the value of SBOM name from metadata.component.name field from SBOM json
+	And namespace column should be empty
+	And group column should contain the value of metadata.component.group field from SBOM json
+	And version column should contain the value of metadata.component.version field from SBOM json
+	And package reference column should contain the value of components.purl from SBOM json
+	And license column should be empty
```

Comment:
Which license column do you reffer here? there are 3 lisence columns

Reply:
updated